api/v1alpha1/agent/openapi.yaml

```diff
@@ -53,6 +53,42 @@ paths:
             application/json:
               schema:
                 $ref: '../openapi.yaml#/components/schemas/Error'
+  /api/v1/agents/{id}:
```

Comment:
Should this be `/api/v1/agents/{id}/status`?

Reply:
it can be. I wrote it like this cause we don't have another put operation on the agent. 
I'll change it